Jenkinsfile

```diff
@@ -423,7 +423,7 @@ pipeline {
       steps {
         script {
           parallel(cambpmGetMatrixStages('engine-webapp-unit', failedStageTypes, { allowedStageLabels, dbLabel ->
-            return cambpmWithLabels(allowedStageLabels.minus('cockroachdb')) && cambpmWithLabels(cambpmGetDbType(dbLabel))
+            return cambpmWithLabels(allowedStageLabels.minus('cockroachdb'), cambpmGetDbType(dbLabel))
```

Comment:
Do we need the `.minus('cockroachdb')` with the new changes?

Reply:
Yes, we do.